### ISSUE TITLE
`WRONG_INDENTATION` rule incorrectly formatting custom getters when `extendedIndentForExpressionBodies` is `true`

### ISSUE THREAD

**Issue by 0x6675636b796f75676974687562** [saveourtool/diktat#1587]
When `extendedIndentForExpressionBodies` is `true`, _Diktat_ expects _properties_ to be formatted as follows:

```kotlin
val a: Int =
        42

val b: Int
    get() =
            42
```

The above is **perfectly consistent**, yet **incompatible** with how IDEA formats _properties_ with _custom getters_, even with `CONTINUATION_INDENT_FOR_EXPRESSION_BODIES` flag set:

```kotlin
val b: Int
    get() =
        42
```

The workaround is to either remove the linebreak:

```kotlin
val b: Int
    get() = 42
```

... or convert the expression body to a block body:

```kotlin
val b: Int
    get() {
        return 42
    }
```

We may need to fine-tune the way _Diktat_ treats the `extendedIndentForExpressionBodies` flag.

## Environment information
* diktat version: 1.2.4.2
